src/components/Charts/SmallTimeSeries.tsx

```diff
@@ -32,22 +32,26 @@ interface Props {
   unitSystem: UnitSystem
   /** Data type being displayed */
   data_type: string
+  /** Earliest date to show */
+  startTime?: Date
+  /** Maximum date to show */
+  endTime?: Date
 }
 
 /**
  * Small time series chart component (as seen on current conditions page)
  */
 class SmallTimeSeriesChartBase extends React.Component<Props, object> {
   public render() {
-    const { name, softMax, softMin, timeSeries, unitSystem, data_type } = this.props
+    const { name, softMax, softMin, timeSeries, unitSystem, data_type, startTime, endTime } = this.props
```

Comment:
Codacy found an issue: [Identifier 'data_type' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6263729)